packages/next/telemetry/project-id.ts

```diff
@@ -28,13 +27,6 @@ function _getProjectIdByGit() {
   }
 }
 
-export function getProjectId(): string {
-  const projectId =
-    _getProjectIdByGit() || process.env.REPOSITORY_URL || process.cwd()
-
-  // Compute a one-way, salted hash of the project ID. This value cannot be
-  // reversed.
-  // See the top of the file for a more detailed reason why we ask for this
-  // data.
-  return computeHash(projectId.trim().toLowerCase()) || ''
+export function getRawProjectId(): string {
+  return _getProjectIdByGit() || process.env.REPOSITORY_URL || process.cwd()
```

Comment:
Why are we no longer using `computeHash` here?

Reply:
The hashing is now handled within the `Telemetry` class using a method called `oneWayHash`. This specific helper was renamed/behavior changed to be the "raw" variant to avoid implementation details leaking into it.